util/string-util.js

```diff
@@ -3,6 +3,11 @@ function isCaseInsensitiveEqual(source, target) {
     if (source.toLowerCase() === target.toLowerCase()) return true;
 }
 
+const isUndefined = (p) => {
```

Comment:
this is a one line function, can we just use "p === undefined" each time instead of calling a function?